tests/infra_test.go

```diff
@@ -366,6 +370,53 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", func() {
 		)
 	})
 
+	Describe("tls configuration", func() {
+
+		It("should be propagated to virt-api, virt-handler and virt-exportpoxy", func() {
+			cipher := tls.CipherSuites()[0]
+			kvConfig := util.GetCurrentKv(virtClient).Spec.Configuration.DeepCopy()
+			kvConfig.TLSConfiguration = &v1.TLSConfiguration{
+				MinTLSVersion: v1.VersionTLS12,
+				Ciphers:       []string{cipher.Name},
+			}
+			tests.UpdateKubeVirtConfigValueAndWait(*kvConfig)
+			newKv := util.GetCurrentKv(virtClient)
+			Expect(newKv.Spec.Configuration.TLSConfiguration.MinTLSVersion).To(BeEquivalentTo(v1.VersionTLS12))
+			Expect(newKv.Spec.Configuration.TLSConfiguration.Ciphers).To(BeEquivalentTo([]string{cipher.Name}))
+
+			tests.EnableFeatureGate("VMExport")
```

Comment:
Let's move everything up to this point to set up.

Reply:
Done, thanks!